lib/signin.ts

```diff
@@ -12,8 +12,13 @@ export default async function signin(email: string, password: string) {
     email: email,
     password: password,
   });
-
-  await setToken(token);
+  try {
+    const error = JSON.parse(token) as BackendError;
+    alert(error.errorMessage.toString());
```

Comment:
These types of things should be done in frontend. Check https://github.com/team-jasmijn/frontend/blob/fffb948edfba8b210c83edeb8a7a7ff6f8eb3695/screens/SignUpScreen.tsx#L43-L60